tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -95,6 +86,47 @@ def test_fsx_lustre(
         storage_capacity=storage_capacity,
     )
     cluster = clusters_factory(cluster_config)
+    _test_fsx_lustre(cluster, region, scheduler, os, mount_dir, bucket_name, storage_type=None, auto_import_policy=None)
+
+
+@pytest.mark.regions(["eu-west-1"])
+@pytest.mark.instances(["c5.xlarge", "m6g.xlarge"])
+@pytest.mark.schedulers(["slurm"])
+@pytest.mark.usefixtures("instance")
+# FSx is not supported on CentOS 6
+@pytest.mark.skip_oss(["centos6"])
+# FSx is only supported on ARM instances for Ubuntu 18.04 and Amazon Linux 2
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "alinux", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "centos7", "*")
+@pytest.mark.skip_dimensions("*", "m6g.xlarge", "ubuntu1604", "*")
```

Comment:
minor: these aren't needed because they're covered in _add_unsupported_arm_dimensions in confest_markers.py

Reply:
I'm just going to leave them for now and then I'll remove all these annotations in a future PR